projects/optic/src/commands/api/add.ts

```diff
@@ -253,6 +261,18 @@ export const getApiAddAction =
       );
       process.exitCode = 1;
       return;
+    } else if (config.vcs?.type === VCS.Git && config.vcs.status === 'dirty') {
```

Comment:
I don't understand this case, why not let them add an untracked API when in a git repo?